src/components/clusters/wizards/rosa/CIDRScreen/CIDRFields.jsx

```diff
@@ -39,6 +39,7 @@ function CIDRFields({
   const { getFieldProps, getFieldMeta, setFieldValue, validateForm } = useFormState();
 
   const isFieldDisabled = isDefaultValuesChecked || disabled;
+  const isHypershift = formValues.hypershift === 'true';
```

Comment:
refrain from hard-coding form keys; use the corresponding `FieldId` constant for the hypershift form value, instead.

Reply:
(i see now that's just code moved, and wasn't introduced with this PR, but it might be worth fixing while we're at it)